### PR TITLE
PR for homepage

### PR DESCRIPTION
**Pull request by narharikale**
I have added the following features
- hero image
- shop by category
- new arrivals

ecom-pr-homepage

### CODE REVIEW

File: homepage.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                        <a href="#" class="categories-media-overlay">
+                            <h2 class="color-white position-center">Women</h4>
+                            <!-- <button class="btn btn-outline bg-transperent font-size-regular color-white position-center">Shop Now</button> -->
+                        </a>
+                    </div>
+                    <div class="categories-media">
+                        <img class="responsive-img obj-fit-cover br-top-sm" src="/assets/img/kid1.jpg" alt="no img">
+                        <a href="#" class="color-white categories-media-overlay">
+                            <h2 class="position-center">Teen</h2>
+                            <!-- <button class="btn btn-outline bg-transperent font-size-regular color-white position-center">Shop Now</button> -->
```
Comment (shraddha15-creator) [inline homepage.html:75]: remove commented code

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            </div>
+        </section>
+        <section class="container" >
+               <h2 class="font-size-md mr-top-md" >Shop by categories</h2> 
+                <div class="categories-container">
+                    <div class="categories-media">
+                        <img class="responsive-img obj-fit-cover br-top-sm" src="/assets/img/man1.jpg" alt="no img">
+                        <a href="#" class="categories-media-overlay">
+                           <h2 class="color-white position-center">Men</h2>
+                            <!-- <button class="btn btn-outline bg-transperent font-size-regular color-white position-center">Shop Now</button>   -->
```
Comment (shraddha15-creator) [inline homepage.html:61]: remove commented code if not needed

**Review by shraddha15-creator — APPROVED** [review]
Looks good

**Review by saketkum11 — APPROVED** [review]

File: homepage.html (hunk 1 above)
Comment (narharikale) [reply homepage.html:75]: thanks for pointing out 👍
but i kept it for purpose

**Review by mohammadkhizerkhan — APPROVED** [review]

**Pull request merged**